src/api/apiHandler.js

```diff
@@ -0,0 +1,17 @@
+export const apiHandler = async (endpoint, method = 'GET', requestBody) => {
```

Comment:
you should also handle errors here probably

Reply:
done